src/tests/api/cars-axios-cookies.spec.js

```diff
@@ -0,0 +1,105 @@
+import { expect } from '@playwright/test';
+import { wrapper } from 'axios-cookiejar-support';
+import { CookieJar } from 'tough-cookie';
+
+import { test } from '../../fixtures/test.fixtures';
+import {DEFAULT_BRANDS_RESPONSE_BODY} from "../fixtures/brands.js";
+import {DEFAULT_BRAND_MODELS} from "../fixtures/models.js";
+import axios from "axios";
+import {FIRST_USER_CAR_RESPONSE_BODY} from "../fixtures/cars.js";
+import {config as testConfig} from "../../../config/config.js";
+import {USER} from "../../data/users.js";
+test.describe("API requests with axios and cookies jar", ()=> {
+    /* HOMEWORK IMPLEMENTATION */
+    let client;
+
+    test.beforeAll(async() => {
+        const jar = new CookieJar();
+        client = wrapper(axios.create({
+            baseURL: testConfig.apiURL,
+            jar,
+            validateStatus: status => {
+                return status < 501
+            }
+        }));
+
+        await client.post('/auth/signin', {
+            "email": USER.email,
+            "password": USER.password,
+            "remember": false
+        });
+
+    });
+    test("Check the FIRST car of a current user", async ({userApiClient}) => {
+        const userFirstCarId = FIRST_USER_CAR_RESPONSE_BODY.data.id;
```

Comment:
it should not be a fixture

Reply:
fixed within https://github.com/AndreyTests/Playwright/pull/4